doc/source/dsintro.rst

```diff
@@ -366,13 +366,32 @@ dictionary.
 
 .. _basics.dataframe.from_series:
 
-From a Series
-~~~~~~~~~~~~~
+From a Panda Series
+~~~~~~~~~~~~~~~~~~~
 
 The result will be a DataFrame with the same index as the input Series, and
 with one column whose name is the original name of the Series (only if no other
 column name provided).
 
+.. ipython:: python
+
+   df = pd.DataFrame()
+   df
+   x1 = pd.Series([1,2,3,4], index=[1,2,3,4])
+   x1
+   df['A'] = x1
+   df
+   x2 = pd.Series([5,5], index=[1, 3])
+   x2
+   df['B'] = x2
```

Comment:
This is more an example of how to add a column to existing dataframe, and how the values get aligned first before getting added. 
So I think it is not fully fitting for here (this section is only about creating a DataFrame)

Reply:
Okay, I will update that.

Reply:
Is there a place I could document
- Adding/replacing a column from a numpy series with more than 1 columns
- Adding/replacing multiple columns from a numpy series with multiple columns.

Please reject the pull request.

Reply:
There is already a section in this file about adding columns: http://pandas.pydata.org/pandas-docs/stable/dsintro.html#column-selection-addition-deletion